### ISSUE TITLE
Templates

### ISSUE THREAD

**Issue by Christilut** [JuoCode/vscode-surround#1]
Would be nice to be able to define templates that the plugin uses.

**Comment by stealthgyro** [comment]
Yeah, I like to do try, catch, finally in a lot of scenarios as well.
